### DISCUSSION TITLE
FDS Extract Surface

### DISCUSSION THREAD

**Discussion by protonspot** [firemodels/fds#12660] [Q&A]
Is there any possible way to check if my extract vent is extracting the flow at the rate I specified for the extract surface?

Context: I have huge smoke accumulation in one part of my building (a dead-end corridor) which shouldnt be. The extract vents activate totally fine as per the Controls specified but I am just not sure if they are extracting out the smoke at the specified extract rate. Do I need to put in a device or a slice file to verify that?

**Comment by rmcdermo** [comment]
Yes, you should look at the solid phase output quantities in the user guide.  Apply the DEVC to the surface you are interested in.  Practice with a simple case to convince yourself you are applying things correctly.

**Reply by protonspot** [reply]
I did check the boundary quantities and DEVC but I just couldnt come across something that can help me specifically check the extract rate of my vent. 

The case is pretty complex in this scenario, however, it has already been run with success for natural vent. Just putting an extract mechanical vent is making smoke accumulation and I am confused as to determine if my extract vent is performing as it should?

**Reply by rmcdermo** [reply]
You are not considering my suggestion to create a simple case first to understand how to convince yourself the DEVC is giving you the correct results.

Look at simple cases like [mass_heat_wall_device_test](https://github.com/firemodels/fds/blob/master/Verification/Flowfields/mass_heat_wall_device_test.fds) but you will want to consider 'TOTAL MASS FLUX', not just advective.

**Reply by protonspot** [reply]
Okay, I will run a simple test case and update here with the results.
